.github/workflows/docker-build-and-publish.yml

```diff
@@ -43,7 +53,7 @@ jobs:
 
   docker:
     needs: prepare
-    runs-on: ${{ contains(matrix.platform, 'arm64') && 'ARM64' || 'ubuntu-latest' }}
+    runs-on: ${{ contains(matrix.platform, 'arm64') && github.repository == 'games-on-whales/gow' && 'ARM64' || 'ubuntu-latest' }}
```

Comment:
I had to make this check since I don't have any self-hosted runners.

But do you need this actually? Is it for performance reasons? From my experience using qemu (which this workflow already does) the performance is quite alright when building arm64 images. My personal docker images all use an amd64 runner with the qemu action to build for other platforms and it works fine.

Reply:
If we'd remove the platform split we could completely remove the merge jobs at the end, making this workflow a little less complex.